apps/ledger-live-desktop/src/renderer/hooks/useStoryly.tsx

```diff
@@ -0,0 +1,15 @@
+function useStoryly() {
+  const script = document.createElement("script");
+
+  script.src = "https://web-story.storyly.io/v2/storyly-web.js";
+  script["custom-element"] = "storyly-web";
+  script.async = true;
+
+  document.body.appendChild(script);
+
+  return () => {
+    document.body.removeChild(script);
+  };
+}
+
+export default useStoryly;
```

Comment:
I'm not a fan of `export default` stuff, i prefer exporting with a name :3 (but that's me - and except UI components because it's already all over the place for components in our repos)